CWE-94-Python_darwin/greed2411_Flask-YouTube/ydl.py

```
@@ -0,0 +1,103 @@
+    id_generated = id_generator()
+    try:
+
+        if url == "":
+
+            pass
+
+        else:
+
+            if choice == 1:
+
+                subprocess.call(
```

Ruff (0.13.2)

`subprocess` call with `shell=True` identified, security issue

(S602)